scripts/prebuildify.js

```diff
@@ -0,0 +1,93 @@
+const versionChanged = require('version-changed')
+const prebuildify = require('prebuildify')
+const path = require('path')
+const tar = require('tar')
+const abi = require('node-abi')
+const gh = require('ghreleases')
+
```

Comment:
This file basically does:

1. checks whether we should build or not (if version in `package.json` has changed)
1. if so, runs `prebuildify` which creates (in the linux case) `./prebuilds/linux-x64/{node,electron}-napi.node`
1. bundles the binaries in the previous step into a `.tar.gz` file
1. if needed, creates a release on github for this particular version
1. uploads the `.tar.gz` file to the release